tests/framework/jailer.py

```diff
@@ -247,7 +247,7 @@ def setup(self, use_ramdisk=False):
             )
             utils.run_cmd(cmd)
 
-        if self.netns:
+        if self.netns and self.netns not in utils.run_cmd('ip netns list')[1]:
```

Comment:
`utils.run_cmd('ip netns list')[1]` Contains the entire list of net namespaces ? Or just the first one ?

Reply:
From the [man page](https://man7.org/linux/man-pages/man8/ip-netns.8.html): 

> ip netns list - show all of the named network namespaces

              This command displays all of the network namespaces in
              /var/run/netns

`run_cmd()` returns a tuple of (return code, stdout, stderr). This is why we compare against result[1], which is `stdout`.